src/aap_eda/services/ruleset/activation_podman.py

```diff
@@ -145,20 +141,25 @@ def run_worker_mode(
                 f"command: {args}"
             )
 
-            models.ActivationInstance.objects.filter(
-                id=activation_instance.id
-            ).update(
-                status=ActivationStatus.RUNNING,
-                updated_at=timezone.now(),
-                activation_pod_id=container.id,
+            now = timezone.now()
+            activation_instance.status = ActivationStatus.RUNNING
+            activation_instance.updated_at = now
+            activation_instance.activation_pod_id = container.id
+            activation_instance.save(
+                update_fields=["status", "updated_at", "activation_pod_id"]
             )
-            models.Activation.objects.filter(
-                id=activation_instance.activation.id,
-                is_valid=False,
-            ).update(
-                status=ActivationStatus.RUNNING,
-                is_valid=True,
-                modified_at=timezone.now(),
+
+            activation = activation_instance.activation
+            activation.status = ActivationStatus.RUNNING
+            activation.is_valid = True
+            activation.status_updated_at = now
+            activation.save(
+                update_fields=[
+                    "status",
+                    "status_updated_at",
+                    "is_valid",
+                    "modified_at",
```

Comment:
when save with `update_fields` we need to pick explicitly `modified_at` even though we don't need to provide a value.

Reply:
Indeed, I wrongly supposed that it would work OTB. Anyway, taking care of updating these fields seems to me a bit error prone. Isn't it still better to centralize this logic in the model?